apps/public-reference/pages/applications/review/confirmation.tsx

```diff
@@ -45,7 +45,10 @@ export default () => {
             {t("application.review.confirmation.whatExpectFirstParagraph.attend")}
             {/* TODO: url slug seems to be not completed */}
             {listing?.urlSlug && (
-              <Link href={`/${listing.urlSlug}`}>
+              <Link
+                href={`listing/id=${listing.id}`}
+                as={`/listing/${listing.id}/${listing.urlSlug}`}
+              >
                 {t("application.review.confirmation.whatExpectFirstParagraph.listing")}
```

Comment:
Not caused by this change, but we should fix: Next.js links are supposed to wrap their content in `<a>` tags ([docs](https://nextjs.org/docs/api-reference/next/link))

Reply:
Good point, I'll fix before we merge this.